apps/api/src/dotenvcreate.mjs

```diff
@@ -0,0 +1,105 @@
+import { SecretsManagerClient, GetSecretValueCommand } from '@aws-sdk/client-secrets-manager';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import { resolve, dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+console.time('dotenvcreate');
+
+const secretName = process.argv[2];
```

Comment:
I suggest using actual CLI arguments instead of positional items. It's more future proof.

Reply:
Ok